components/link-social/css/stories/default.stories.mdx

```diff
@@ -6,7 +6,7 @@ import "../index.scss";
 
 <Meta
   id="css-link-social"
-  title="CSS Component/Link to Social Media"
+  title="CSS Component/Link Social"
```

Comment:
Isn't "Link to Social Media" more helpful to readers of the documentation than "Link Social"?

I would prefer to keep the long title.

We might even put the `list-social` under `"Link to Social Media/Link list"`